web/list.jsp

```diff
@@ -282,8 +285,8 @@ Click <a href="<%= rawPath %>">download <%= basename %></a><%
                                 // SRCROOT is read with UTF-8 as a default.
                                 r = IOUtils.createBOMStrippedReader(in,
                                     StandardCharsets.UTF_8.name());
-                                AnalyzerGuru.writeDumpedXref(
-                                        request.getContextPath(), a, r, out,
+                                AnalyzerGuru.writeXref(
```

Comment:
This was not reverted after you found the true problem.

Reply:
Yeah, I was sort of waiting for you to chime in :-) Need to test if the fix still works with `writeDumpedXref()`.

Reply:
I can see that `writeDumpedXref()` is just a wrapper of `writeXref()` so it should work fine.